apps/mudita-center-e2e/src/specs/news/news-check-online.e2e.ts

```diff
@@ -44,16 +44,14 @@ describe("News Page Check", () => {
     await lastNewsCardCommunityLinkText.scrollIntoView()
     await expect(lastNewsCardCommunityLinkText).toBeDisplayedInViewport()
   })
-  // TODO: to fix
   it.skip("Verify News Cards", async () => {
```

Comment:
```suggestion
  it("Verify News Cards", async () => {
```

Reply:
I didn't notice it sorry [42cddc3](https://github.com/mudita/mudita-center/pull/1901/commits/42cddc3e5c2943b748cec05709d0fbc14968748c)